docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -139,6 +139,9 @@ Refer to the [advanced SSL configuration guide](./ssl.md) for additional details
 | `RESTAPI_HOST`            | [Internal](#notes-on-host-names-and-port-numbers) host name of the `restapi` application.                                              | `modeler-restapi`                                                | -             |
 | `RESTAPI_PORT`            | [Internal](#notes-on-host-names-and-port-numbers) port number on which the `restapi` serves the regular API endpoints.                 | `8081`                                                           | `8081`        |
 | `RESTAPI_MANAGEMENT_PORT` | [Internal](#notes-on-host-names-and-port-numbers) port number on which the `restapi` serves the management API endpoints.              | `8091`                                                           | `8091`        |
+| `http_proxy`              | Specifies the proxy server to be used for outgoing HTTP requests.                                                                      | `http://proxy.example.com:8080`                                  | -             |
+| `https_proxy`             | Specifies the proxy server to be used for outgoing HTTPS requests.                                                                     | `https://secureproxy.example.com:443`                            | -             |
+| `no_proxy`                | A comma-separated list of domain names or IP addresses for which the proxy should be bypassed.                                         | `localhost,127.0.0.1,.example.com`                               | -             |
```

Comment:
You could maybe put the proxy-related env vars into their own section.

Reply:
Maybe we should also add a note on why they are lowercase.

Reply:
Made both changes.